recipes/spglib/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "spglib" %}
+{% set version = "1.9.8.11" %}
+{% set sha256 = "94f8860e77fcdfe882adc38dde8c67e4789c82aa3d04ecfac0aa7a5e912d70a5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  skip: true  # [win and (py27 or py34)]
+  number: 0
+  script: 
+    - python setup.py install --single-version-externally-managed --record record.txt 
+
+requirements:
+  build:
+    - python
+    - numpy
```

Comment:
Unfortunately, numpy's ABI is a special feature of conda (like Python versions). I don't know why the `x.x` is used instead of `1.11` or some other concrete number, but I imagine that it has to do with the fact that conda recipes are templates that are filled in.  Others probably know more.